### PR TITLE
Fix version of postgres image

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Motivation and context
Our workflows fail (e.g. [1](https://github.com/opencv/cvat/actions/runs/4956755413?pr=6075), [2](https://github.com/opencv/cvat/actions/runs/4956806970?pr=5892), [3](https://github.com/opencv/cvat/actions/runs/4956552204)) due to [problem with new version of postgres:15-alpline image](https://github.com/docker-library/postgres/issues/1076).

This PR fix the version of postgres image

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 services:
   cvat_db:
     container_name: cvat_db
-    image: postgres:15-alpine
+    image: postgres:15-alpine3.17
```
Comment (azhavoro) [inline docker-compose.yml:8]: @kirill-sizov  
could you add a comment why this change is needed?

Comment (kirill-sizov) [reply docker-compose.yml:10]: Added
